README.md

```diff
@@ -3,34 +3,43 @@
 API plugins, microservices, and commands for accessing NetCDF data/metadata contained in iRODS data objects.
 
 Currently supported version(s) of iRODS:
-   - 4.2.8
+   - 4.2.12
+   - 4.3.1
 
 OS Distributions presently supported:
+* on iRODS 4.2.12
    - Centos 7
-   - Ubuntu 16.04
    - Ubuntu 18.04
+* on iRODS 4.3.1
+   - AlmaLinux 8 (RHEL 8)
+   - Centos 7
+   - Debian 11
+   - Debian 12
+   - Rocky Linux 9 (RHEL 9)
+   - Ubuntu 18.04
+   - Ubuntu 20.04
+   - Ubuntu 22.04
 
 Automated building and packaging is available as before, but is now done through CMake.
 
-## Building the Plugin for iRODS 4.2+
+## Building the Plugin for iRODS 4.3.1
 
 Make sure the development packages for OpenSSL, NetCDF, HDF5, and iRODS itself are installed on the build machine, as well
 as these build externals:
 
-   - `irods-externals-cmake3.11.4-0`
-   - `irods-externals-clang6.0-0`
-   - `irods-externals-boost1.67.0-0`
+   - `irods-externals-cmake3.21.4-0`
+   - `irods-externals-clang13.0.0-0`
+   - `irods-externals-boost1.81.0-0`
```

Comment:
Let's avoid putting the version numbers in the README.

The `find_package` call for iRODS 4.3.1 declares the required versions.

See the GenQuery2 and HTTP API repo's README for examples.